MarkBind/markbind
# Set search bar width to placeholder text

**What is the purpose of this pull request?**



Resolves #665 

**Overview of changes:**
Binds the width of the search bar to the length of the placeholder text so as to avoid clipping of the placeholder text.

Before:
![search bar before](https://user-images.githubusercontent.com/48304907/177129262-bd2b01c4-7604-4a46-88d5-57c778a89384.png)

After:
![search bar after](https://user-images.githubusercontent.com/48304907/177129303-efb6a89a-44fa-47b9-bfbe-3b5d88297cb1.png)


**Anything you'd like to highlight/discuss:**
The width is set using the `ch` unit, which is defined according to the width of the `o` character in a given typeface. Hence, it might run into issues with certain non-standard typefaces, should the MarkBind typeface be changed in the future.

**Testing instructions:**
Use a long placeholder text to check if the width of the search bar expands accordingly.


**Proposed commit message: (wrap lines at 72 characters)**
Set search bar width to placeholder text



---

**Checklist:** :ballot_box_with_check:
